builtin/providers/aws/resource_aws_cloudwatch_event_rule.go

```diff
@@ -218,7 +222,7 @@ func buildPutRuleInputStruct(d *schema.ResourceData) *events.PutRuleInput {
 		input.Description = aws.String(v.(string))
 	}
 	if v, ok := d.GetOk("event_pattern"); ok {
-		pattern, _ := normalizeJsonString(v.(string))
+		pattern, _ := normalizeJsonString(v)
```

Comment:
We cannot really return error from this function, due to the signature expecting us to return `*events.PutRuleInput` type. Unless you have a suggestion.